cvmfs/util.h

```diff
@@ -38,6 +38,12 @@ const int kDefaultFileMode = S_IWUSR | S_IRUSR | S_IRGRP | S_IROTH;
 const int kDefaultDirMode = S_IXUSR | S_IWUSR | S_IRUSR |
                             S_IRGRP | S_IXGRP | S_IROTH | S_IXOTH;
 
+enum CatalogProperty {
```

Comment:
This is very neat.  The tri-state property type should be part of sql.(cc|h), I think.  @reneme, thoughts?